src/dodal/devices/i18/table.py

```diff
@@ -0,0 +1,28 @@
+from dataclasses import dataclass
+
+from ophyd_async.core import (
+    AsyncStatus,
+    StandardReadable,
+)
+from ophyd_async.epics.motor import Motor
+
+
+@dataclass
```

Comment:
How does this differ from the RasterStage?

Reply:
it's not different really

Reply:
there are two stages, of sorts.

Reply:
deleted